metasv/generate_sv_intervals.py

```diff
@@ -186,39 +201,49 @@ def find_other_bp(aln, isize_mean, isize_sd, svtype, soft_clip_location, num_sd=
                 other_bp = max(soft_clip_location - (abs(aln.tlen) + isize_mean - 2*dist_R_end),0)
             elif dist_R_end <= min_dist_end and not aln.is_reverse and aln.pos <= aln.pnext:
                 other_bp = max(soft_clip_location + (abs(aln.tlen) + isize_mean - 2*dist_L_end),0)
-            #if other_bp == -1:
-            #    print aln, dist_L_end, dist_R_end, aln.is_reverse, aln.pos > aln.pnext, other_bp
-            return other_bp
+        elif not skip_neigh:
+            if not aln.is_reverse and aln.pos > aln.pnext and aln.aend < (soft_clip_location + wiggle):
```

Comment:
could probably simplify this chain of conditionals. the max with 0 could always happen at the end.